scripts/database.js

```diff
@@ -249,3 +249,15 @@ const database = {
     },
   ],
 };
+
+export const getSchools = () => {
+  return database.schools.map(schools => ({ ...schools }))
+}
+
+export const getArchetypes = () => {
+  return database.archetypes.map(archetypes => ({ ...archetypes}))
```

Comment:
The iterator should be singular, not plural, as it represents one archetype object in the archetype array.